CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 1.48.1 (2019-07-19)
+
+- [Use Python3 for Python in interactive setup](https://github.com/serverless/serverless/pull/6406)
+- [Fixing broken link for Node install.](https://github.com/serverless/serverless/pull/6405)
+- [Added Cloud Build option for serverless deploy guide](https://github.com/serverless/serverless/pull/6401)
+- [Changed AWS subscription filters to use function object name](https://github.com/serverless/serverless/pull/6402)
+- [https://github.com/serverless/serverless/pull/6408](https://github.com/serverless/serverless/pull/6408)
```

Comment:
Changelog is now not complete.. and this one misses description

Reply:
updated